skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -202,6 +202,8 @@ expect enum class SkikoKey {
     KEY_NUMPAD_DECIMAL;
 }
 
+expect val SkikoKey.platformKeyCode: Long
```

Comment:
Can its return type be Int?

Long in k/js is boxed into an object, while Int is a primitive type, which might be better when used a lot.

Is this function intended to be used from Compose?

Reply:
Unfortunately, Int also be converted to Long anyway, because it will passed to 
```
expect value class Key(val keyCode: Long)
``` (https://github.com/JetBrains/androidx/blob/5f442ff2a5ea30cc8d49888554a31d20c66c9dce/compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/key/Key.kt#L30)

Yeah, it intended to be used in Compose.

Reply:
rename to platformKeyCode, and save Int type.
But add check in iOS, to be sure that iOS platformKeyCode not bigger than Int.MAX_VALUE